src/sidebars/docs.json

```diff
@@ -533,6 +533,10 @@
                 "name": "Product manual",
                 "url": "/docs/experiments/manual"
             },
+            {
+                "name": "Advanced: Running experiments without feature flags",
```

Comment:
```suggestion
                "name": "Running experiments without feature flags",
```

Reply:
I'd like to include the term "Advanced", since it not something most users will do, and helps guide our users by giving them an opinion that they'll likely not need this